doc/scheduler-job-examples.tex

```diff
@@ -20,6 +20,7 @@ \subsection{Example Job Script: Fluent}
 \texttt{slurm-249.out}), and/or by examining the ``moment'' file in 
 \texttt{/disk/nobackup/<yourjob>} (hint: it starts with your job-ID) on the node running
 the job. \textbf{Caveat:} take care with journal-file file paths.
+\newpage
```

Comment:
Please let LaTeX manage new pages. It's OK the figures float; they aren't too far.

Reply:
fixed